### DISCUSSION TITLE
Is this project still actively maintained?

### DISCUSSION THREAD

**Discussion by jrsmith3** [mssun/passforios#652] [General]
It looks like the last stable release was about two years ago. Does anyone know if this application is still being maintained? If not, is there an alternative? Thanks.

**Comment by SimplyDanny** [comment]
I honestly don't think so. I have contributed a lot in the past. But when the response time got longer and longer I lost my motivation as well.

I like the app and the concept of how passwords are managed by the `pass` approach generally. It's a pity that this project is now abandoned.

**Comment by adadnc** [comment]
I'm actively using this app and I would be thankful to see that this app is maintained and improved.

**Comment by jrsmith3** [comment]
Do you think there is enough interest in the community to attempt to maintain this app? I'm not volunteering (yet), I am trying to see if ongoing development can be made sustainable.

**Reply by SimplyDanny** [reply]
Interest is definitely there. As far as I know, this is the only app offering a Pass companion on iOS. About motivation to contribute I cannot say much. Back then there wasn't much traffic in the repository.

Keeping the app up and running and fixing some bugs shouldn't be a big deal. Supporting the whole zoo of different encryption variants people were constantly requesting and fixing issues in this area is a whole other story, though.

**Reply by jrsmith3** [reply]
You don't happen to have any kind of admin rights on this repo, do you?

**Reply by arildhof** [reply]
Just want to say I'm actively using this app as well. Thanks to the original author and all contributors. I hope this project can continue to be maintained and evolve!

**Reply by SimplyDanny** [reply]
> You don't happen to have any kind of admin rights on this repo, do you?

No, I don't have any admin rights.

**Reply by simlay** [reply]
Has anyone reached out to @mssun? According to their LinkedIn, it looks like they've got a new job that's less focused on open source contributions - not faulting them I go in and out of open source work from job to job. There's also an email address listed on their GitHub profile/website.

**Reply by jrsmith3** [reply]
@simlay i emailed @mssun about a week ago. I have not heard back yet.

I agree with you: I'm not faulting them for not keeping this project up to date either -- maintaining open source projects is quite difficult as I understand it.

**Reply by jrsmith3** [reply]
I think if we could get maybe three or four devs that would be interested in volunteering, we could talk about forking this app.

Full disclosure: I am not a swift developer; I do engineering management for some data engineering teams. I am pretty decent at python fwiw.

**Reply by mssun** [reply]
Hi there, thanks for following the updates of the app. I'll spend some time during this holiday season to do some maintenance. Maybe figure put a way for a healthy & sustainable community.

**Reply by jrsmith3** [reply]
Hi @mssun, thank you for checking in on this thread! I would be happy to help organize community development around this app. Feel free to reach out.

**Reply by SimplyDanny** [reply]
I also offer my help to update tools and CI to bring the project back to a buildable state. I'd need write access for that though. 😅

**Reply by mssun** [reply]
> I also offer my help to update tools and CI to bring the project back to a buildable state. I'd need write access for that though. 😅

Added you as the collaborator. Thanks!

I'm trying to build from scratch and found some issues :(

**Comment by SimplyDanny** [comment]
I managed to bring everything into a state, where the project builds and CI is able to run all tests.

The project is now entirely based on Swift Package Manager (SPM) except for the GopenPGP library. Linting and formatting now also run as SPM plugins. With all that, I was able to remove some scripts and build steps as well as the dependency on Carthage. This simplifies the build process significantly. At least Xcode 15.3 and hence at least macOS 14 are required for development.

The [deploy step](https://github.com/mssun/passforios/actions/runs/12076475354) seems to fail due to access issues. Perhaps you can have a look into that, @mssun.

**Reply by mssun** [reply]
Great! I have fixed the deploying pipeline and release a TestFlight (0.15.0 build 3).

**Comment by jrsmith3** [comment]
I'm going to close this thread because my original question has been answered. Thanks!
